releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -2,6 +2,45 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0 - September 24, 2024
+
+### Enhancements
+
+#### Variant Feature Flags
```

Comment:
BTW, I think it will be helpful to call out that the variant feature flags require v8 of the provider library to work if users load feature flags from App Configuration.